test/test_browser.py

```diff
@@ -5490,8 +5489,9 @@ def test_audio_worklet_post_function(self, args):
     '': ([],),
     'closure': (['--closure', '1', '-Oz'],),
   })
+  @requires_sound_hardware
   def test_audio_worklet_modularize(self, args):
-    self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMODULARIZE=1', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html')] + args)
+    self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMODULARIZE=1', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html'), '-DTEST_AND_EXIT'] + args)
```

Comment:
Revert the `-DTEST_AND_EXIT` part of this change

Reply:
I'm just doing so (and moving it to the other PR). 😀